code/modules/materials/material_recipes.dm

```diff
@@ -17,8 +17,11 @@
 		. += new recipe_type(src)
 
 /decl/material/proc/generate_recipes(var/reinforce_material)
-	. = list()
 
+	if(phase_at_stp() != MAT_PHASE_SOLID)
```

Comment:
Why not just pass a temperature argument in? If someone's standing in subzero temperatures they might be able to craft a frozen cake batter reinforced window, for example, even if they can't at STP.

Reply:
Right. To do that we'd need to rewrite a lot anyway, so how it is in the PR is fine.